### DISCUSSION TITLE
Delta apply job failed on uploading project

### DISCUSSION THREAD

**Discussion by Notramus** [opengisch/QField#3341] [Q&A]
I am having trouble locating the problem in our latest changes which leads to the failed "Delta apply job". 

If I manually reapply the changes via the "changes" tab in QFIELD Cloud it changes to pending and later to error again. 

Looking at the jobs it seems that the uploading part is failing: 
09:18:57.313 root      INFO     Local files list for project "a10e242d-4083-4b92-a850-0e47d9939b94":
Name                                        Size  MD5 Checksum
--------------------------------------  --------  --------------------------------
DeleteLater.qmd                             1056  bbfe08e8abf4aedf75e377c692d78a5e
DeleteLater.sqlite                       6684672  f047f06ef05717e403e74f10ffcf10df
Freiburg_2022.qgs                        1616832  072a31dfc36e84235f9795875b1afa8c
Hintergrund/Hintergrund.qmd                  672  da362a0f0e67483b58de35320614f36c
Hintergrund/Hintergrund.sqlite          79015936  db43dccd0e5af4d0f807748d352affb4
Kontrolle noch notwendig.dbf                 229  777a48d24938b2b6e2ff3825f8f4f7d3
Kontrolle noch notwendig.prj                 145  c742bee3d4edfc2948a2ad08de1790a5
Kontrolle noch notwendig.shp                 112  876cde9302b982ff13f7fe5ad5d8283c
Kontrolle noch notwendig.shx                 108  3b6c7ad7fbcf586281feeb71b3c3449c
Kontrolle/Kontrolle.qmd                     1057  ef0fea7fb26306c6cf93c3593b9a8600
Kontrolle/Kontrolle.sqlite              51527680  578356a12131d07010565a20d0faa1a3
Monitoring/01_03.csv                       13805  3693092907163aa10abec35354a817cc
Monitoring/Monitoring_2022.sqlite        6893568  68c2e355dbcce39e95666458fe2b0d5d
Monitoring/csv/Kontrolle 1_2_3_4_5.csv     21482  7160a69001c01531d9977aafeb832c3a
Personal/Sonstiges.sqlite                6742016  1c29e53ee28665a72989af363ce29440
Vermutete Ausbreitung.dbf                    110  34d072cd807024408821d54264bd6b4b
Vermutete Ausbreitung.prj                    145  c742bee3d4edfc2948a2ad08de1790a5
Vermutete Ausbreitung.shp                    728  d9310da33c4406598b9d24c026d5e692
Vermutete Ausbreitung.shx                    132  f9a0ff226df159315e8a856204f69070
delete.qmd                                   672  da362a0f0e67483b58de35320614f36c
delete.sqlite                           63950848  b06c55ee355feba1851aeee5860e2908

Does anyone have any idea?

**Comment by suricactus** [comment]
Thanks for pointing this out. The reason was that jobs had a very low limit of 60 seconds before getting timed out. Now the timeout is set to 10 minutes and all should be good if you rerun the job apply.

**Reply by Notramus** [reply]
Thanks for the help! I have tried reapplying the changes, but it still failed after a few minutes?

**Reply by suricactus** [reply]
Can you try again now?

**Reply by Notramus** [reply]
Everything working as intented now! Thanks a lot Suricactus :)
